spooky-doodle/Bangazon-Website
# Jp 4

Select a new category when selling a new product. Use the sell something link in the nav-bar 

Fixes #4 #13 

## Type of change

Please delete options that are not relevant.


# Testing Instructions
 try selling something and check your database to see if the POST request worked.
also try setting the price greater than 10,000r


# Checklist:
